doc/developer-guide/api/functions/TSRemap.en.rst

```diff
@@ -72,8 +72,10 @@ invoked by current and all previous still used configurations. This is an option
 
 :func:`TSRemapPostConfigReload` is called to indicate the end of the the new remap configuration
 load. It is called on the newly and previously loaded plugins, invoked by the new, current and
-previous still used configurations. It also indicates if the configuration reload was successful
-by passing :macro:`TS_SUCCESS` or :macro:`TS_ERROR`. This is an optional entry point.
+previous still used configurations. It also indicates wheather the configuration reload was successful
+by passing :macro:`TSREMAP_CONFIG_RELOAD_FAILURE` in case of failure and to notify the plugins if they
+are going to be part of the new configuration by returning :macro:`TSREMAP_CONFIG_RELOAD_SUCCESS_PLUGIN_USED`
```

Comment:
"returning"? Shouldn't that be passed *to* the plugin?

Reply:
right, passed to plugin.